idd/Energy+.idd.in

```diff
@@ -70888,10 +70888,26 @@ LoadProfile:Plant,
        \type real
        \units m3/s
        \ip-units gal/min
-  A5 ; \field Flow Rate Fraction Schedule Name
+  A5 , \field Flow Rate Fraction Schedule Name
        \required-field
        \type object-list
        \object-list ScheduleNames
+  A6 , \field Plant Loop Fluid Type
+       \required-field
```

Comment:
If these added inputs are optional for steam, then I doubt this should be a required field.